src/Tizen.NUI/src/public/Size2D.cs

```diff
@@ -56,6 +56,11 @@ public Size2D(int x, int y) : this(Interop.Vector2.new_Vector2__SWIG_1((float)x,
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        internal Size2D(float x, float y) : this(Interop.Vector2.new_Vector2__SWIG_1(x, y), true)
```

Comment:
Hello, I am not sure so I am just asking...
1) this(Interop.Vector2.new_Vector2__SWIG_1(x, y), true)
2) this(Interop.Vector3.new_**Vector3**__SWIG_1(x, y, **0.0f**), true)
which one is right?